src/openPr.js

```diff
@@ -53,6 +53,33 @@ const addArtifact = async (inputs, releaseId) => {
   return artifact
 }
 
+const createDraftRelease = async (inputs, newVersion) => {
+  const run = runSpawn()
+  try {
+    const releaseCommitHash = await run('git', ['rev-parse', 'HEAD'])
+
+    logInfo(`Creating draft release from commit: ${releaseCommitHash}`)
+
+    const { data: draftRelease } = await callApi(
+      {
+        method: 'POST',
+        endpoint: 'release',
+        body: {
+          version: newVersion,
+          target: releaseCommitHash,
+        },
+      },
+      inputs
+    )
+
+    logInfo(`Draft release created successfully`)
+
+    return draftRelease
+  } catch (err) {
+    core.setFailed(`Unable to create draft release: ${err.message}`)
```

Comment:
are we sure this stops the execution immediately? because otherwise we're catching the error and execution continues, even if just for a short time